doc/related_projects.rst

```diff
@@ -148,8 +148,8 @@ and tasks.
 - `nolearn <https://github.com/dnouri/nolearn>`_ A number of wrappers and
   abstractions around existing neural network libraries
 
-- `keras <https://github.com/fchollet/keras>`_ Deep Learning library capable of
-  running on top of either TensorFlow or Theano.
+- `Keras <https://www.tensorflow.org/api_docs/python/tf/keras>`_ High-level API for
```

Comment:
Actually there is now https://github.com/keras-team/keras so maybe this will change in the near future. Also I'm not sure about "scikit-learn inspired API" part. There are certainly some inspiration, but the API is generally not compatiblke with scikit-learn without `tf.keras.wrappers.scikit_learn` wrappers.

Reply:
That is their old repository, which may be used again at some point, but it is unclear when. For now I think the best place to link is the docs.

Happy to change the description if you have any other suggestions. The Keras authors have stated that it was inspired by sklearn, and it's certainly closer than pure Pytorch (not that that's a fair comparison).

Reply:
OK, fair enough. Thanks!